opts/opts.go

```diff
@@ -188,3 +188,35 @@ func ValidateLabel(val string) (string, error) {
 	}
 	return val, nil
 }
+
+// ValidateSysctl validates an sysctl and returns it.
+func ValidateSysctl(val string) (string, error) {
+	validSysctlMap := map[string]bool{
+		"kernel.msgmax":          true,
```

Comment:
would you consider adding `kernel.perf_event_paranoid` and `kernel.kptr_restrict` to the whitelist?

as a use case: i'm working on a proof of concept around flame graphs, visualizing perf event data in a node.js app following along the work here https://gist.github.com/trevnorris/9616784.

have had a devil of a time sorting out getting these kernel parameters to stick in a running container. came upon your PR, and am stoked to see a `--sysctl` flag in the works.